src/Form/Form.js

```diff
@@ -121,9 +121,14 @@ class Form {
         this.form.querySelectorAll(FIELD_SELECTOR).forEach(input => this.addInput(input))
     }
 
-    // TODO: try to filter down to only submit buttons
     get submitButtons () {
-        return this.form.querySelectorAll(SUBMIT_BUTTON_SELECTOR)
+        return [...this.form.querySelectorAll(SUBMIT_BUTTON_SELECTOR)]
+            .filter((button) => {
+                const content = button.textContent
+                const ariaLabel = button.getAttribute('aria-label')
+                const title = button.title
+                return !/password|show|toggle|reveal|hide/i.test(content + ariaLabel + title)
+            })
```

Comment:
Here I am trying to exclude the little buttons to show and hide passwords.

Reply:
Please add a comment for this 👍🏻